schemas/v0.26.0/minimalpermissionsguidanceplugin.schema.json

```diff
@@ -8,6 +8,14 @@
     },
     "apiSpecsFolderPath": {
       "type": "string"
+    },
+    "permissionsToExclude": {
```

Comment:
Since we're just adding this property, we can't update previous versions schemas because the property isn't available there. Let's only add it to the latest version

Reply:
fixed